sympy/polys/domains/algebraicfield.py

```diff
@@ -362,6 +366,58 @@ def from_GaussianRationalField(K1, a, K0):
         """Convert a GaussianRational element 'a' to ``dtype``. """
         return K1.from_sympy(K0.to_sympy(a))
 
+    def _do_round_two(self):
+        from sympy.polys.numberfields.basis import round_two
+        ZK, dK = round_two(self.ext.minpoly, radicals=self._nilradicals_mod_p)
+        self._maximal_order = ZK
+        self._discriminant = dK
+
+    def maximal_order(self):
+        """
+        Get an integral basis for the field, as an Order.
+
+        See Also
+        ========
+
+        integral_basis()
+
+        """
+        if self._maximal_order is None:
+            self._do_round_two()
+        return self._maximal_order
+
+    def integral_basis(self):
+        r"""
+        Get an integral basis for the field.
+
+        If the field is of degree $n$ and $\theta$ is a primitive element, then
+        the basis is returned in the form of an upper triangular $n \times n$
+        matrix over domain :ref:`QQ`, whose column vectors represent the basis
+        elements as linear combinations over $\{1, \theta, \ldots, \theta^{n-1}\}$.
+
+        See Also
+        ========
+
+        maximal_order()
+
+        """
+        ZK = self.maximal_order()
+        return ZK.QQ_matrix
+
+    def discriminant(self):
+        """Get the discriminant of the field."""
```

Comment:
I'm having some trouble with the Sphinx build. Was wondering if you could offer some advice.

I have added some text to the docstring for the `AlgebraicField` class (defined in `sympy.polys.domains.algebraicfield`), which says:

```
    The maximal order, or ring of integers, of the field can also be
    obtained from the :py:meth:`~.maximal_order` method, as a
    :py:class:`~sympy.polys.numberfields.modules.Submodule`.
```
but Sphinx can't seem to resolve the
```
:py:class:`~sympy.polys.numberfields.modules.Submodule`
```
reference at the end. It says,
```
docstring of sympy.polys.domains.algebraicfield.AlgebraicField:171: WARNING: py:class reference target not found: sympy.polys.numberfields.modules.Submodule
```

It's already an absolute path, i.e. starts from `sympy....` so I don't know how I can help it to find this class. Does the class have to be imported into the module where this docstring lives? Because I can't do that, since it causes a circular import.

Have you run into this before? Thanks.

Reply:
I'm not sure. Maybe just `~.Submodule`?

Easiest way to get help with debugging is to push what you have to the PR.